pkg/routes/url.go

```diff
@@ -22,6 +24,12 @@ const (
 	AddUserUrl       = "/add_user"
 )
 
+var (
+	CreateKwdUrlFull = fmt.Sprintf("%s/:kw_type", CreateKwdUrl)
+	EditKwdUrlFull   = fmt.Sprintf("%s/:id/:kw_type", EditKwdUrl)
+	DeleteKwdUrlFull = fmt.Sprintf("%s/:id/:kw_type", DeleteKwdUrl)
+)
```

Comment:
why these can't be constants as upper ones?

Reply:
You can't assing the result of a function call to a const:
![image](https://user-images.githubusercontent.com/28830303/205582614-05aef818-d596-4f55-9e6c-91aaa1213e49.png)